x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_add_to_case_actions.tsx

```diff
@@ -79,16 +91,16 @@ export const useAddToCaseActions = ({
     [activeStep, isTourShown]
   );
 
-  const createCaseFlyout = casesUi.hooks.getUseCasesAddToNewCaseFlyout({
+  const createCaseFlyout = casesUi.hooks.useCasesAddToNewCaseFlyout({
     onClose: onMenuItemClick,
-    onSuccess,
+    onSuccess: onCaseSuccess,
```

Comment:
never thought about this before, but is there a handler for failure or errors?

Reply:
Hey @michaelolo24! No there is not. Cases handle that behind the scenes. If it is useful we put a handler for errors on another PR.